src/sonic-config-engine/translate_acl

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+import openconfig_acl 
+import pyangbind.lib.pybindJSON as pybindJSON
+
+import sys
+import json
+
+def dump_json(filename, data):
+    with open(filename, 'w') as outfile:
+        json.dump(data, outfile, indent=4, sort_keys=True, separators=(',', ':'))
+
+def generate_rule_json(table_name, rule):
+    rule_idx = rule.config.sequence_id
+    rule_props = {}
+    rule_data = {}
+    rule_data["ACL_RULE_TABLE:"+table_name+":Rule_"+str(rule_idx)] = rule_props
+    rule_data["OP"] = "SET"
+    
+    rule_props["priority"] = 10000 - rule_idx
+    if rule.actions.config.forwarding_action == "ACCEPT":
+        rule_props["PACKET_ACTION"] = "FORWARD"
+    elif rule.actions.config.forwarding_action == "DROP":
+        rule_props["PACKET_ACTION"] = "DROP"
+    elif rule.actions.config.forwarding_action == "REJECT":
+        rule_props["PACKET_ACTION"] = "DROP"
+    else:
+        print "Unknown rule action %s in table %s, rule %d!" % (rule.actions.config.forwarding_action, table_name, rule_idx)
+        return {}
+    
+    if rule.ip.config.protocol == "":
```

Comment:
compare with "" does not look like a good practice. check below

http://stackoverflow.com/questions/9573244/most-elegant-way-to-check-if-the-string-is-empty-in-python